aruco_pose_estimation/aruco_pose_estimation/pose_estimation.py

```
@@ -42,7 +42,7 @@ def pose_estimation(rgb_frame: np.array, depth_frame: np.array, aruco_detector:
     # corners, marker_ids, _ = cv2.aruco.detectMarkers(frame, aruco_dict_type, parameters=parameters)
 
     # new code version
-    corners, marker_ids, rejected = aruco_detector.detectMarkers(image=rgb_frame)
+    corners, marker_ids, rejected = cv2.aruco.detectMarkers(image=rgb_frame, dictionary=aruco_dict, parameters=aruco_params)
```

Ruff (0.14.14)

[warning] Unpacked variable `rejected` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)